### DISCUSSION TITLE
Does frame generation works with Sunshine?

### DISCUSSION THREAD

**Discussion by SHChen1997** [LizardByte/community#432] [Sunshine]
Hi, I'm wondering whether frame generation (DLSS 3) works with Sunshine?

When I turn on DLSS 3, it seems the the frame rate increases on the host side, but not on the client (moonlight)

Without DLSS 3:
Host fps : around 50
Client incoming fps from network : around 50

With DLSS 3:
Host fps : around 80
Client incoming fps from network : around 50

Host :
GPU : RTX4070Ti
Windows : Windows 10 22H2
Nvidia driver : 552.12
Sunshine : 0.23.0

Client : Nvidia shield tv pro 2019
Moonlight (android) : 12.1

Host and Client are connected through wired 1 GbE

Appreciate for any help, thanks!

**Comment by Travillion** [comment]
Did you set Moonlight to max 50 fps in the stream settings?

**Reply by SHChen1997** [reply]
I set the fps in moonlight to 60, and I also test that the client can get solid 60 fps if host fps is above 60 without DLSS 3
(I test it by lower the texture quality, the host fps will be above 60 fps without DLSS 3, then moonlight can get solid 60 incoming fps from network)

**Reply by Travillion** [reply]
Just grasping at straws here.... did you disable HAGS in sunshine?

**Reply by SHChen1997** [reply]
The HAGS is enabled in sunshine

**Comment by Travillion** [comment]
Then I am at a loss to help, I'm sorry.

**Reply by SHChen1997** [reply]
lol thank you for the help anyway!

[a comment by an automated account is not shown]
